### PR TITLE
Videos: Fix 0 views and empty license field

### LINKED ISSUE #3589: [Bug]  I cant see the youtube view count when i use any of the invidious frontend variants.

### PR DESCRIPTION
**Pull request by techmetx11**
Videos with Standard YouTube license have an empty metadata object

Closes #3589

### CODE REVIEW

File: src/invidious/views/watch.ecr [hunk 1]
```diff
@@ -181,7 +181,11 @@ we're going to need to do it here in order to allow for translations.
                 <% end %>
             </p>
             <% if video.license %>
-                <p id="license"><%= translate(locale, "License: ") %><%= video.license %></p>
+                <% if video.license == "" %>
```
Comment (SamantazFox) [inline src/invidious/views/watch.ecr:184]: the use of `.empty?` is preferred here!

**Review by SamantazFox — APPROVED** [review]

**Comment by unixfox** [comment]
@SamantazFox I think the tests may need to be adapted?

**Comment by SamantazFox** [comment]
> @SamantazFox I think the tests may need to be adapted?

Oh, right, the mocks needs to be updated!

**Review by SamantazFox — APPROVED** [review]

**Comment by SamantazFox** [comment]
I'm marking this as blocked, until I've updated [invidious/mocks](https://github.com/iv-org/mocks)

**Comment by SamantazFox** [comment]
Sorry for the time it took. I've rebased your changes and updated the mocks and the spec.
You'll have to use `extract_text` (from `src/invidious/yt_backend/extractors_utils.cr`) to extract the views count, as the scheduled live streams & premiered videos still use the old structure (See failing spec).

**Comment by SamantazFox** [comment]
Thanks for your contribution :)

**Pull request merged**